imports/plugins/included/product-variant/containers/variantFormContainer.js

```diff
@@ -195,6 +199,44 @@ const wrapComponent = (Comp) => (
       Meteor.call("products/updateProductField", variant._id, "isVisible", !variant.isVisible);
     }
 
+    /**
+     * @method updateInventoryPolicyIfChildVariants
+     * @description update parent inventory policy if variant has children children
+     * @param {Object} variant product or variant document
+     * @return {undefined} return nothing
+     */
+    updateInventoryPolicyIfChildVariants = (variant) => {
+      // Get all siblings, including current variant
+      const options = ReactionProduct.getSiblings(variant);
+      // Get parent
+      const parent = ReactionProduct.getVariantParent(variant);
+
+      // If this is not a top-level variant, update top-level inventory policy as well
+      if (parent) {
```

Comment:
Could we combine this into one conditional e.g. `if (parent && options && options.length)` ?